t/14-isotovideo.t

```diff
@@ -186,6 +186,21 @@ subtest 'error handling when loading test schedule' => sub {
     };
 };
 
+subtest 'load test success when there is no PRODUCTDIR' => sub {
+    chdir($pool_dir);
+    path(bmwqemu::STATE_FILE)->remove if -e bmwqemu::STATE_FILE;
+    path('vars.json')->remove         if -e 'vars.json';
+    mkdir('opensuse')                                                   unless -e 'opensuse';
+    symlink("$data_dir/tests/lib", "opensuse/lib")                      unless -e 'opensuse/lib';
+    mkdir('opensuse/products')                                          unless -e 'opensuse/products';
+    mkdir('opensuse/products/foo')                                      unless -e 'products/foo';
+    symlink("$data_dir/tests/tests", "opensuse/tests")                  unless -e 'opensuse/tests';
+    symlink("$data_dir/tests/needles", "opensuse/products/foo/needles") unless -e 'opensuse/products/foo/needles';
```

Comment:
please use single tickets in strings where you do not want to evaluate any variables

```suggestion
    symlink("$data_dir/tests/lib", 'opensuse/lib')                      unless -e 'opensuse/lib';
    mkdir('opensuse/products')                                          unless -e 'opensuse/products';
    mkdir('opensuse/products/foo')                                      unless -e 'products/foo';
    symlink("$data_dir/tests/tests", 'opensuse/tests')                  unless -e 'opensuse/tests';
    symlink("$data_dir/tests/needles", 'opensuse/products/foo/needles') unless -e 'opensuse/products/foo/needles';
```

Reply:
fixed